kcidb/monitor/spool/__init__.py

```diff
@@ -12,7 +12,8 @@
 import email
 import email.policy
 from google.cloud import firestore
-from kcidb.misc import ArgumentParser, iso_timestamp, log_and_print_excepthook
+from kcidb.argparse import ArgumentParser
+from kcidb.misc import iso_timestamp, log_and_print_excepthook
```

Comment:
Actually, `iso_timestamp` needs to be in `kcidb.argparse` as well, since it's acting (and is documented) as an argparse "argument type".